expansion/src/main/java/no/unit/nva/expansion/model/ExpandedPublishingRequest.java

```diff
@@ -162,10 +157,37 @@ private static ExpandedPublishingRequest createRequest(PublishingRequestCase dat
         entry.setFinalizedBy(dataEntry.getFinalizedBy());
         return entry;
     }
-    
+
     private static Publication fetchPublication(PublishingRequestCase publishingRequestCase,
                                                 ResourceService resourceService) {
         return attempt(() -> resourceService.getPublicationByIdentifier(
-            publishingRequestCase.extractPublicationIdentifier())).orElseThrow();
+                publishingRequestCase.extractPublicationIdentifier())).orElseThrow();
+    }
+
+    private static ExpandedTicketStatus getExpandedTicketStatus(PublishingRequestCase publishingRequestCase) {
+        switch (publishingRequestCase.getStatus()) {
+            case PENDING:
+                return getNewTicketStatus(publishingRequestCase);
+            case COMPLETED:
+                return ExpandedTicketStatus.COMPLETED;
+            case CLOSED:
+                return ExpandedTicketStatus.CLOSED;
+        }
+        return null;
+    }
+
+    private static ExpandedTicketStatus getNewTicketStatus(PublishingRequestCase publishingRequestCase) {
+        if (isNull(publishingRequestCase.getAssignee())) {
+            return ExpandedTicketStatus.NEW;
+        } else {
+            return ExpandedTicketStatus.PENDING;
+        }
+    }
+
+    private TicketStatus getTicketStatusParse() {
+        if (!this.getStatus().equals(ExpandedTicketStatus.NEW)) {
```

Comment:
Should we consider returning TicketStatus New if ExpandedTicketStatus is Pending? I see that the toTicketEntry() only is used for testing, maybe we should move it.

Reply:
Ja, toTicketEntry() is used only in one test. If we return status=New, I am getting error as we don't want this status new outside the expansion.

Reply:
Like Haakon commented: "_if toTicketEntry() is only used in testing, move the code to the test.
This so the production code shouldn't have functionality it doesn't need -> like decompressing after an expansion._"